### PR TITLE
Refresh packages and template

### PR DESCRIPTION
**Pull request by benjay10**
I was using this service to write some reports, and I noticed that certain syntax features where not working because the JavaScript version was too old. Time for an update?

This PR bumps the template version to 1.6.0 (not latest, but already an improvement) and bumps all packages to their latest version.

I don't think this would cause an issue with the scripts that are already in use. If so, those stacks are still free to use a version of this service before the 1.6.0 upgrade.

### CODE REVIEW

**Review by lagartoverde — APPROVED** [review]

**Comment by lagartoverde** [comment]
To be honest it's been a really long time since I used this so you might want to wait for the review from Felix before merging, but it seems good to me

**Comment by benjay10** [comment]
Felix also approved and testing a couple reports showed no issues.

**Pull request merged**
